### ISSUE TITLE
Add metric that matches the duration in access log 

### ISSUE THREAD

**Issue by jobergum** [vespa-engine/vespa#26410]
Due to https://github.com/vespa-engine/vespa/issues/26408, the `duration` logged in the [access log](https://docs.vespa.ai/en/access-logging.html) can be considerably higher than `query_latency`, also because the access log duration field includes the time it takes to render the response. 

I think that we should have a metric that corresponds 100% with the `duration` logged to the access log, bonus for percentile calculations.

**Comment by yngveaasheim** [comment]
Agreed, and the `container.handled.latency.sum` metric with the correct `handler` tag should be a much closer match here. There is no percentiles for this, though, but I believe we should rather use histograms for than percentiles going forwards.

**Comment by bjorncs** [comment]
Resetting priority. This is part of larger effort to evaluate all existing container metrics.
